checker/src/org/checkerframework/checker/nullness/NullnessAnnotatedTypeFactory.java

```diff
@@ -154,6 +154,13 @@ public NullnessAnnotatedTypeFactory(BaseTypeChecker checker, boolean useFbc) {
                 org.checkerframework.checker.nullness.compatqual.MonotonicNonNullType.class,
                 MONOTONIC_NONNULL);
 
+        // Alias deprecated annotation
```

Comment:
I wasn't suggesting anything, just pointing why Travis is crashing.  

We don't copy values for aliased type annotations. Some checkers, such as the Upper Bound Checker, override `AnnotatedTypeFactory#aliasedAnnotation` to copy the values over.

Instead of aliasing, it might be easier to check for both annotations in DefaultTypeHierarchy.